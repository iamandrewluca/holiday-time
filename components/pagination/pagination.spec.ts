import { describe, it, expect } from '@jest/globals'
import { render } from '@testing-library/vue'
import Pagination from './pagination.component.vue'

describe('Pagination', () => {
  it('should render and match snapshot', () => {
    const { baseElement } = render(Pagination, { props: { page: 1 } })
    expect(baseElement).toMatchSnapshot()
  })
})
