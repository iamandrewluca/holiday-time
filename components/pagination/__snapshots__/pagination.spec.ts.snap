// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination should render and match snapshot 1`] = `
<body>
  <div>
    <div
      class="inline-flex justify-even space-x-3"
      data-testid="pagination"
    >
      <button
        class="bg-yellow-300 text-yellow-900 rounded-lg py-2 px-5"
      >
        <svg
          class="w-6 h-6"
          fill="none"
          stroke="currentColor"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12.066 11.2a1 1 0 000 1.6l5.334 4A1 1 0 0019 16V8a1 1 0 00-1.6-.8l-5.333 4zM4.066 11.2a1 1 0 000 1.6l5.334 4A1 1 0 0011 16V8a1 1 0 00-1.6-.8l-5.334 4z"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
          />
        </svg>
      </button>
       
      <span
        class="bg-yellow-300 text-yellow-900 rounded-lg py-2 px-5"
      >
        
    1
  
      </span>
       
      <button
        class="bg-yellow-300 text-yellow-900 rounded-lg py-2 px-5"
      >
        <svg
          class="w-6 h-6"
          fill="none"
          stroke="currentColor"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M11.933 12.8a1 1 0 000-1.6L6.6 7.2A1 1 0 005 8v8a1 1 0 001.6.8l5.333-4zM19.933 12.8a1 1 0 000-1.6l-5.333-4A1 1 0 0013 8v8a1 1 0 001.6.8l5.333-4z"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
          />
        </svg>
      </button>
    </div>
  </div>
</body>
`;
